src/main/java/com/mojang/brigadier/exceptions/BuiltInExceptionProvider.java

```diff
@@ -28,6 +28,8 @@
 
     DynamicCommandExceptionType readerInvalidEscape();
 
+    DynamicCommandExceptionType readerInvalidTerm();
```

Comment:
Might be better to not prefix it with `reader` since it is not the reader which parses terms (it only parses strings).

Reply:
agreed. fixed.